src/SFA.DAS.AssessorService.Application.Api/Controllers/ExternalApi/ExternalApiController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.AssessorService.Api.Types.Models.ExternalApi.Learners;
+using SFA.DAS.AssessorService.Application.Api.Properties.Attributes;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Api.Controllers.ExternalApi
+{
+    [Authorize(Roles = "AssessorServiceInternalAPI")]
```

Comment:
I'm not sure this is correct?

Reply:
Yeah it's correct. Have a look at the other controllers :)